packages/react-native-editor/ios/GutenbergDemo.xcodeproj/project.pbxproj

```diff
@@ -667,7 +667,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
This seems like an odd change. Based upon https://github.com/WordPress/gutenberg/commit/e980666f5ff3423cc88a5e8394fd35ef39b41419, it would appear this was performed by the release script. Any ideas as to why this occurred? 

Given its a part of the Demo app and references simulators, I would imagine it would have no negative impact on the release build.